build.assets/makefiles/base/docker/registry.mk

```diff
@@ -1,7 +1,7 @@
 .PHONY: all
 
 REPODIR=$(GOPATH)/src/github.com/docker/
-VER=v2.6.0
+VER=v2.7.1-gravitational
```

Comment:
Is this version backwards-compatible with 2.6? IIRC last time we upgraded the registry minor version (2.4 -> 2.6 I think) we had to upgrade vendored library too b/c tele build was producing incompatible images. Am I correct @a-palchikov?

Reply:
According to the 2.7.1 release notes, they fixed an issue when going to 2.6.0 -> 2.7.1, but besides that I don't know if there might be more compatibility issues lurking.